src/commands/deploy.ts

```diff
@@ -230,4 +242,34 @@ export default class Deploy extends Command {
       await this.runRemote();
     }
   }
+
+  async validate_graph(graph: DependencyGraph): Promise<void> {
+    cli.action.start(chalk.blue('Validating deployment'));
+    const { data: validation_results } = await this.app.api.post<ValidationResult[]>(`/graph/validation`, graph);
```

Comment:
Lets make this non-blocking in a non-200 response. Someone without internet should still be able to do a local deploy.

Reply:
Done 👍